clab/config.go

```diff
@@ -321,24 +335,24 @@ func (c *CLab) verifyLinks() error {
 }
 
 // VerifyImages will check if image referred in the node config
-// either pullable or present or is available in the local registry
-// if it is not available it will emit an error
+// either pullable or is available in the local image store
 func (c *CLab) VerifyImages(ctx context.Context) error {
-	images := map[string]struct{}{}
+
+	images := make(map[string]string)
+
 	for _, node := range c.Nodes {
-		// skip image verification for bridge kinds
-		if node.Config().Kind == "bridge" || node.Config().Kind == "ovs-bridge" {
-			continue
-		}
-		if node.Config().Image == "" {
-			return fmt.Errorf("missing required image for node %s", node.Config().ShortName)
+
+		for _, imageName := range node.GetImages() {
+			if imageName == "" {
+				log.Errorf("missing required image for node %q", node.Config().ShortName)
+			}
+			images[imageName] = node.GetRuntime().GetName()
 		}
 
-		images[node.Config().Image] = struct{}{}
 	}
 
-	for image := range images {
-		err := c.Runtime.PullImageIfRequired(ctx, image)
+	for image, runtimeName := range images {
+		err := c.Runtimes[runtimeName].PullImageIfRequired(ctx, image)
```

Comment:
are we always sure that `c.Runtimes[runtimeName]` always exist and is not `nil` ?

Reply:
Yes, in this case `runtimeName` comes from `node.GetRuntime().GetName()`, so this means runtime is already there (since it responds to `GetName`)
And `nodes` all have a runtime thanks to this 
```go
err = n.Init(nodeCfg, nodes.WithRuntime(c.GlobalRuntime()), nodes.WithMgmtNet(c.Config.Mgmt))
```